pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ryzenadj-controller"
+version = "1.0.0"
+authors = [
+    { name="Derek J. Clark", email="derekjohn.clark@gmail.com" },
+]
+description = " Python module that provides a userspace interface for RyzenAdj using Unix Sockets "
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: POSIX :: Linux",
+]
+dependencies = []
+
+[project.urls]
+"Homepage" = "https://github.com/ShadowBlip/ryzenadj-controller"
+"Bug Tracker" = "https://github.com/ShadowBlip/ryzenadj-controller/issues"
+
+[tool.setuptools.packages.find]
+include = ["ryzenadj_controller"]
+
+[project.scripts]
+ryzenadj-controller = "ryzenadj_controller.ryzenadj-controller:main"
```

Comment:
needs to be `ryzenadj_controller.ryzenadj_controller:main"` sencond _ is - isntead.